src/routes/game/components/dialogs/components/ChooseWhetherToCounterDialog.vue

```diff
@@ -27,12 +27,7 @@
         </p>
         <div v-if="target" id="target-wrapper">
           <span id="target-icon-wrapper" class="d-flex justify-center align-center">
-            <v-icon
-              size="x-large"
-              color="red"
-              icon="mdi-target" 
-              aria-hidden="true"           
-            />
+            <v-icon size="x-large" color="red" icon="mdi-target" aria-hidden="true" />
```

Comment:
This is causing the lint failure. We have Vue/max-attributes-per-line set to 3. Please roll this back to the previous format. It looks like you may have a local editor config conflicting with the project lint config